cypress/e2e/article.cy.js

```diff
@@ -1,13 +1,17 @@
 describe('', () => {
+  let user;
   before(() => {
-
+    cy.task('generateUser').then((generatedUser) => {
+      user = generatedUser;
+    });
   });
 
-  it('', () => {
-
-  });
-
-  it('', () => {
-
+  it('should provide an ability to create the article', () => {
+    cy.visit('/user/register');
```

Comment:
If you're creating a user via a request, there is no need to go to the specific page.

Reply:
fixed